docs/guide/hosting.md

```diff
@@ -28,19 +28,23 @@ MongoDB requires a minimum of 5GB free disk space at all times. Small websites m
 
 Deployment processes will vary depending on the hosting environment, technical requirements, and team preferences. There are a few steps that all deployment processes should include.
 
-1. If Apostrophe has been running on the server already, **stop the application process(es).**
-2. **Run the data migration task: `node app @apostrophecms/migration:migrate`.** This will run both core data migrations as well as project-level migrations.
-3. **Run the build process: `node app @apostrophecms/asset:build`.** This runs automatically in development environments, but not if `NODE_ENV=production` or if the `autoBuild: false` option is set on the application. That is to ensure that the build is an intentional part of deployment and that developers can execute it along with other production build tasks as necessary.
+1. **Run the data migration task: `node app @apostrophecms/migration:migrate`.** This will run both core data migrations as well as project-level migrations.
+2. **Run the build process: `node app @apostrophecms/asset:build`.** This runs automatically in development environments, but not if `NODE_ENV=production` or if the `autoBuild: false` option is set on the application. That is to ensure that the build is an intentional part of deployment and that developers can execute it along with other production build tasks as necessary.
+3. If Apostrophe has been running on the server already, **stop the application process(es).**
 4. **Start (or restart) the application process(es).**
 
 ## Best practices
 
+### Set the `NODE_ENV` variable for production
+
+Apostrophe includes performance enhancements when in "production mode." In production environments, or staging environments that replicate production, set the `NODE_ENV` to `production`. Be aware that this will disable the automatic user interface build. See the section above for more about running the build process.
+
 ### Run multiple processes
 
-Running the website on multiple server processes is always a good idea in the production environment. It is usually not necessary to run more than two processes. Also make sure not to run more processes than you have CPU cores available (see your hosting plan).
+Running the website on multiple server processes is always a good idea in the production environment. We recommend running at least two. Also make sure not to run more processes than you have CPU cores available (see your hosting plan).
```

Comment:
Sorry, but we do have to disambiguate a little here still. I don't want people to think they must have two cores, that's a bit expensive and not a requirement. How about:

"You should run at least two processes to guarantee availability if one process is restarting, even if you only have one CPU core. If you have more than two CPU cores, you may run additional processes, one per additional core. If you have a lot of capacity, you might want to reserve a core for MongoDB."

Reply:
works for me.